packages/itwinui-css/src/tabs/tabs.scss

```diff
@@ -2,6 +2,22 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../utils/index';
 
+@mixin iui-tabs-wrapper {
+  display: grid;
+  position: relative;
+
+  .iui-tabs-actions {
+    grid-area: tabs-actions;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
+
+  .iui-tabs-content {
```

Comment:
`iui-tabs-actions` and `iui-tabs-content` don't need to be nested. they can be separate mixins

Reply:
Moved them to separate mixins in [`7c39650` (#860)](https://github.com/iTwin/iTwinUI/pull/860/commits/7c39650b69b80c5de6ec603bd8e862005b0aa25f)